src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -945,75 +945,7 @@ async def __stop(self, job):
         await self.middleware.call('etc.generate', 'pam')
```

Comment:
Might as well clean this up too.
**Imports:**
    Line 18: Remove this line `middlewared.utils.nss.pwd` and `grp` are unused.
    Line 19: Remove this line `NssModule` is unused.

**General Flake8 complaints:**
    Line 676: Missing whitespace after keyword (col 16)

**Typo that should be fixed:**  <----------
    Line 737: Missing 'self'
        (This is the `kinit` function, not sure how it's working!)
        In that same function `krb5ccache` and `ldap_config` are undefined.  Possibly related to the missing `self`.